test/service/test_response.py

```diff
@@ -1,130 +1,41 @@
-                            {
-                                "contact_name": "teal",
-                                "corresponding_contributor": True,
-                                "email": "purple",
-                                "institution": "yellow",
-                                "laboratory": "silver"
-                            }
-                        ],
-                        "publications": [
-                            {
-                                "authors": [
-                                    "green",
-                                    "maroon",
-                                    "gray"
-                                ],
-                                "publication_title": "gray",
-                                "doi": "green",
-                                "pmid": 5331933,
-                                "publication_url": "black"
-                            }
-                        ]
-                    }
-                ],
-                "specimens": [
-                    {
-                        "_type": specimen_value("teal"),
-                        "organism_age": ["purple"],
-                        "organism_age_unit": ["navy"],
-                        "biomaterial_id": specimen_value("6e7d782e-44a2-0d3f-2bf1-337468f62467"),
-                        "disease": ["yellow"],
-                        "id": specimen_value("1cae440e-3be6-ce39-49e9-74721f0066e0"),
-                        "organ": specimen_value("purple"),
-                        "organ_part": specimen_value("black"),
-                        "parent": specimen_value("aqua"),
-                        "biological_sex": ["silver"],
-                        "_source": ["purple"],
-                        "genus_species": ["teal"],
-                        "preservation_method": specimen_value("aqua")
-                    }
-                ],
-                "cell_suspensions": [
-                    {
-                        "organ": ["purple"],
-                        "organ_part": ["black"],
-                        "total_estimated_cells": 5306
-                    }
-                ]
-            },
-            "bundles": [
-                {
-                    "uuid": "cfc75555-f551-ba6c-2e62-0bf0ee01313c",
-                    "version": "2003-08-12T00:52:21"
+    def _get_index(self, index: str, entity_id: str):
+        body = {
+            "query": {
+                "term": {
+                    "entity_id.keyword": entity_id
                 }
-            ],
-            "entity_id": "08d3440a-7481-41c5-5140-e15ed269ea63"
+            }
         }
-        return [ElasticsearchFakeDataLoader.fix_canned_document(entity_type, template)]
+        # Tests are assumed to only ever run with the azul dev index
+        results = self.es_client.search(index=index, body=body)
+        return [results['hits']['hits'][0]['_source']]
+
+    def get_files_index(self, entity_id: str):
```

Comment:
```suggestion
    def _get_file_hits(self, entity_id: str):
```

Likewise for the other two.

Reply:
I just eliminated these helper methods and make entity_type a parameter to get_hits()